client/qiskit_serverless/core/clients/serverless_client.py

```diff
@@ -210,6 +211,7 @@ def jobs(self, **kwargs) -> List[Job]:
         offset = kwargs.get("offset", 0)
         kwargs["offset"] = offset
         status = kwargs.get("status", None)
+        status, _ = _map_status_from_serverless(status)
```

Comment:
Correct me if I'm wrong here @korgan00 . We are not mapping status **FROM** serverless if not **TO** serverless, aren't we?